app/bin/service/analyzer.dart

```diff
@@ -55,6 +56,7 @@ Future _frontendMain(FrontendEntryMessage message) async {
   ));
 
   await withAppEngineAndCache(() async {
+    registerStorageWithRetry();
```

Comment:
Looks like this is a common pattern. Wouldn't it be better to move the `registerStorageWithRetry();` call inside (and in the beginning of) `withAppEngineAndCache`?

Reply:
Yes, and no... it would have to be renamed then :)

I'll leave that for later, as we should refactor how all these components are loaded.